packages/react-refresh/src/ReactFreshRuntime.js

```diff
@@ -465,6 +465,17 @@ export function injectIntoGlobalHook(globalObject: any): void {
       };
     }
 
+    if (hook.isDisabled) {
+      // This isn't a real property on the hook, but it can be set to opt out
+      // of DevTools integration and associated warnings and logs.
+      // Using console['warn'] to evade Babel and ESLint
+      console['warn'](
+        'Something has shimmed the React DevTools global hook (REACT_DEVTOOLS_GLOBAL_HOOK). ' +
+          'Fast Refresh is not compatible with this shim and will be disabled.',
+      );
+      return;
```

Comment:
Adding the console warning wasn't actually disabling Fast Refresh. Needed to add `return` to prevent the rest of the function from overriding `hook` methods.

Reply:
@bvaughn thanks. Skipped my mind.